### DISCUSSION TITLE
用申请的域名管理员登录访问，登录之后为什么一直处于未登录状态？

### DISCUSSION THREAD

**Discussion by starsflows666** [AlistGo/alist#2672] [Q&A]
自己申请的域名，服务器代理也配置了，原地址可以正常进入管理员，用申请的域名登录管理员后台，密码是成功的，登录之后为什么一直处于未登录状态？

**Comment by Dashan-IZ** [comment]
我也遇到了这个问题，请问你们是怎么解决的

**Reply by starsflows666** [reply]
反向代理配置那里修改一下

**Reply by Dashan-IZ** [reply]
这个问题困扰我很久了，希望可以在你这里得到解决

**Reply by starsflows666** [reply]
是要改配置域名对应的地址那里，你现在可以用域名加上5244端口号访问应该是成功的，你试试

**Reply by Dashan-IZ** [reply]
我现在的域名+端口访问不了，因为我配置了反向代理，代理地址就是ip+端口，用ip+端口的方式倒是可以正常登录，我找不出原因在哪里，因为之前还是好的

**Comment by Dashan-IZ** [comment]
starsflows666 ***@***.***>于2022年12月29日 周四00:33写道：

> 反向代理配置那里修改一下
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alist-org/alist/discussions/2672#discussioncomment-4509433>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASUBRJAD4ZMUHZQJEJXGD2LWPRTUXANCNFSM6AAAAAASZ7VXAU>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
请问nginx的反向代理需要修改什么地方

**Comment by chouqiao** [comment]
> 自己申请的域名，服务器代理也配置了，原地址可以正常进入管理员，用申请的域名登录管理员后台，密码是成功的，登录之后为什么一直处于未登录状态？

好像是啥缓存没有清理好，我到第二天它自己好了

**Comment by Dashan-IZ** [comment]
chouqiao ***@***.***>于2022年12月29日 周四10:13写道：

> 自己申请的域名，服务器代理也配置了，原地址可以正常进入管理员，用申请的域名登录管理员后台，密码是成功的，登录之后为什么一直处于未登录状态？
>
> 好像是啥缓存没有清理好，我到第二天它自己好了
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alist-org/alist/discussions/2672#discussioncomment-4529881>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASUBRJAFSKEW3TU2JGZUTK3WPTXURANCNFSM6AAAAAASZ7VXAU>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
我配置了cdn，好像是缓存的锅，好麻烦啊

**Reply by starsflows666** [reply]
好吧

**Comment by Dashan-IZ** [comment]
starsflows666 ***@***.***>于2022年12月29日 周四11:22写道：

> 好吧
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alist-org/alist/discussions/2672#discussioncomment-4530569>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASUBRJG2A2CEUCJXEBDIKDTWPT7XFANCNFSM6AAAAAASZ7VXAU>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
没解决，解决不了，我套了cdn

**Reply by starsflows666** [reply]
我这边是反向代理配置一开始配置错了，你这边我不确定是不是同样的问题，代理名称用申请的域名，目标url要填端口号，发送域名填$host，我是这个配置问题，你看看你这里配置有没有问题

**Reply by chouqiao** [reply]
> 我这边是反向代理配置一开始配置错了，你这边我不确定是不是同样的问题，代理名称用申请的域名，目标url要填端口号，发送域名填$host，我是这个配置问题，你看看你这里配置有没有问题

我也遇到登录不上去的问题，但用的不是宝塔面板，用的是amh面板，反代的参数名字和宝塔的不一样，没目标url和发送域名的说法

**Comment by Dashan-IZ** [comment]
starsflows666 ***@***.***>于2022年12月29日 周四11:29写道：

>
> 我这边是反向代理配置一开始配置错了，你这边我不确定是不是同样的问题，代理名称用申请的域名，目标url要填端口号，发送域名填$host，我是这个配置问题，你看看你这里配置有没有问题
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alist-org/alist/discussions/2672#discussioncomment-4530602>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASUBRJF4Q4RUNQECZLM7JW3WPUAQTANCNFSM6AAAAAASZ7VXAU>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
能把主要配置截个图吗？nginx，

**Reply by starsflows666** [reply]
私聊我帮你看看吧

**Comment by Dashan-IZ** [comment]
starsflows666 ***@***.***>于2022年12月29日 周四11:32写道：

> 私聊我帮你看看吧
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alist-org/alist/discussions/2672#discussioncomment-4530617>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASUBRJFQ5RGBLVE4Y3CGL73WPUA3JANCNFSM6AAAAAASZ7VXAU>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
通过邮件回复你了

**Comment by Dashan-IZ** [comment]
chouqiao ***@***.***>于2022年12月29日 周四11:48写道：

>
> 我这边是反向代理配置一开始配置错了，你这边我不确定是不是同样的问题，代理名称用申请的域名，目标url要填端口号，发送域名填$host，我是这个配置问题，你看看你这里配置有没有问题
>
> 我也遇到登录不上去的问题，但用的不是宝塔面板，用的是amh面板，反代的参数名字和宝塔的不一样，没目标url和发送域名的说法
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alist-org/alist/discussions/2672#discussioncomment-4530673>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASUBRJDRIXXL7WDYTGXMSWDWPUCY7ANCNFSM6AAAAAASZ7VXAU>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
我没有用面板，就是缓存的锅，让我办知道怎样解决他

**Reply by wjn6** [reply]
别人都给你说了是反向代理没设置，你还在这缓存缓存，但凡认真看了文档，都不会问出这种问题

**Comment by Dashan-IZ** [comment]
这个问题我遇到了，这个是反向代理缓存导致的问题，nginx.conf不要配置缓存，如果有嵌套cdn建议把cdn的缓存去掉，试试看，现在我的网站已经正常了，https://specialcloud.top/

**Comment by loohalh** [comment]
同样的问题，我是换了 cdn服务商之后出现的，之前在 cf 一直是好的，换到腾讯后就出现这个问题了，反代设置没问题
